content/release-10-13-0/edge-10-13-0-bundle/10_13_0.md

```diff
@@ -0,0 +1,30 @@
+---
+weight: 60
+title: Release 10.13.0
+layout: redirect
+---
+
+Cumulocity IoT Edge Release 10.13, includes the following improvements, fixes and known issues.
+
+Download the Cumulocity IoT Edge version 10.13 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
+
+### Changing the hostname
+
+You can now change the hostname of your Edge appliance using the user interface. For more information, see [Configuring the hostname using the UI](https://cumulocity.com/guides/edge/configuration/#configuring-the-hostname).
+
+### Configuring the time synchronization
+
+You can now configure the time synchronization of your Edge appliance using the user interface. For more information, [Configuring the time synchronization using the UI](https://cumulocity.com/guides/edge/configuration/#configuring-the-time-synchronization-using-the-ui).
+
+### Downloading the diagnostics
+
+In the previous versions, you could download the diagnostics report using the command line only. This capability has been expanded further. You can now download the diagnostics report using the user interface locally and from your tenant account remotely using the Device Management application. You can also use the REST API to download the report. For more information, see [Downloading diagnostics using the UI](https://cumulocity.com/guides/edge/diagnostics-and-support/#diagnostic-report-through-ui).
+
+### Known issues
+
+|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
+|:---|:---|:---
+|CIE-2962|SSL certificate|**Issue:** Changing the SSL certificate fails. This issue occurs because the ordering of the certificates chain is incorrect.<br><br>**Workaround:** Fix the certificate ordering as described in the [Prerequisites](https://cumulocity.com/guides/edge/installation/#prerequisites) section or generate a self signed certificate. Do not reboot the system before a new certificate has been successfully installed.
+||Installation service|**Issue:** The installation service might return an HTTPS status code 408 error when polling the task in the version 10.11. This is mostly observed during the upgrade.<br><br>**Workaround:** You may ignore the error or poll the `/edge/tasks/latest/update` endpoint until the task is successful. 
```

Comment:
`/edge/tasks/latest/update` > <kbd>/edge/tasks/latest/update<kbd>
We use kbd for endpoints throughout the documentation

Reply:
Done